packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -548,14 +640,24 @@ To edit platform code in an IDE see https://flutter.dev/developing-packages/#edi
 
     if (boolArg('pub')) {
       await pub.get(context: PubContext.create, directory: directory.path, offline: boolArg('offline'));
-      await project.ensureReadyForPlatformSpecificTooling(checkProjects: false);
+      await project.ensureReadyForPlatformSpecificTooling(checkProjects: pluginExampleApp);
```

Comment:
this is confusing, why do we need to only check projects if we're creating an example app

Reply:
makes sense

Reply:
I think the non specificity of "checkProjects" is what's throwing me off (I know you didn't name it).  Like, check what?  If that was a better name, `pluginExampleApp` might make sense.

Reply:
By looking at the code of `ensureReadyForPlatformSpecificTooling`, I also not sure what that meant. @jonahwilliams  do you have any idea? Maybe we can add a TODO comment on that method to either fix the param name or add some doc/comments on the `checkProjects` param?

Reply:
yeah, TODO for now.

Reply:
done